python/pyspark/mllib/tests.py

```diff
@@ -853,6 +853,17 @@ def serialize(l):
         self.assertEqual(sv, serialize(lil.tocsr()))
         self.assertEqual(sv, serialize(lil.todok()))
 
+    def test_convert_to_vector(self):
```

Comment:
We have no Scipy-related tests in `ml` submodule, so I don't add it. If we really need one, please let me know. I can add it quickly.